src/service_name.json

```diff
@@ -0,0 +1,342 @@
+[
+  {
+    "Command": "az account",
+    "AzureServiceName": "Azure CLI",
+    "URL": "https://docs.microsoft.com/azure/cost-management-billing/manage/create-subscription"
+  },
+  {
+    "Command": "az ai-examples",
+    "AzureServiceName": "Microsoft AI",
+    "URL": "https://docs.microsoft.com/archive/msdn-magazine/2017/connect/artificial-intelligence-getting-started-with-microsoft-ai"
+  },
+  {
+    "Command": "az alias",
+    "AzureServiceName": "Azure CLI",
+    "URL": ""
+  },
+  {
+    "Command": "az attestation",
+    "AzureServiceName": "Azure Attestation",
+    "URL": "https://docs.microsoft.com/azure/attestation/overview"
+  },
+  {
+    "Command": "az automation",
+    "AzureServiceName": "Azure Automation",
+    "URL": "https://docs.microsoft.com/azure/automation/automation-intro"
+  },
+  {
+    "Command": "az baremetalinstance",
+    "AzureServiceName": "Microsoft Windows",
+    "URL": "https://docs.microsoft.com/windows-hardware/test/assessments/"
+  },
+  {
+    "Command": "az blockchain",
+    "AzureServiceName": "Azure Blockchain",
+    "URL": "https://docs.microsoft.com/azure/blockchain/"
+  },
+  {
+    "Command": "az blueprint",
+    "AzureServiceName": "Azure Blueprint",
+    "URL": "https://docs.microsoft.com/azure/governance/blueprints/"
+  },
+  {
+    "Command": "az cli-translator",
+    "AzureServiceName": "Azure CLI",
+    "URL": "https://docs.microsoft.com/cli/azure/"
+  },
+  {
+    "Command": "az codespace",
+    "AzureServiceName": "Visual Studio",
+    "URL": "https://docs.microsoft.com/visualstudio/codespaces/overview/what-is-vsonline"
+  },
+  {
+    "Command": "az communication",
+    "AzureServiceName": "Azure Communication Service",
+    "URL": "https://docs.microsoft.com/azure/communication-services/"
+  },
+  {
+    "Command": "az confluent",
+    "AzureServiceName": "None",
+    "URL": ""
+  },
```

Comment:
This is a third-party service in LiftR. Do we just use `None` for the `AzureServiceName`? Will it cuase problems if there are mulitple `None` `AzureServiceName`?

Reply:
Updated